src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -86,6 +87,21 @@ public static Phone parsePhone(String phone) throws ParseException {
         return new Phone(trimmedPhone);
     }
 
+    /**
+     * Parses a {@code String Date_of_birth} into a {@code DateOfBirth}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code Date_of_birth} is invalid.
```

Comment:
I think `Date_of_birth` should be changed to `dob` since that's the actual name of the variable
```suggestion
     * Parses a {@code String dob} into a {@code DateOfBirth}.
     * Leading and trailing whitespaces will be trimmed.
     *
     * @throws ParseException if the given {@code dob} is invalid.
```